src/explore-education-statistics-frontend/src/components/PageSearchFormWithAnalytics.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+import PageSearchForm, {
+  PageSearchFormProps,
+} from '@common/components/PageSearchForm';
+
+const PageSearchFormWithAnalytics = (props: PageSearchFormProps) => {
+  const onSearch = (searchTerm: string) => {
```

Comment:
Inline this

Reply:
Did you mean like this? 
https://github.com/dfe-analytical-services/explore-education-statistics/pull/343/files#diff-8b0c4e30eeaa9e4fbd7e20f768b2c7d8R9

Reply:
Yep indeed :+1: